clients/TypeScript/packages/client/src/ChainSync/ChainSyncClient.ts

```diff
@@ -79,17 +63,27 @@ export const createChainSyncClient = async (options?: {
           },
           requestNext (options) {
             ensureSocketIsOpen(socket)
-            socket.send(JSON.stringify({
-              ...baseRequest,
-              methodname: 'RequestNext',
-              mirror: options?.mirror
-            } as Ogmios['RequestNext']))
+            return requestNext(socket, options)
           },
           shutdown: () => new Promise(resolve => {
             ensureSocketIsOpen(socket)
             socket.once('close', resolve)
             socket.close()
-          })
+          }),
+          startSync: async (points, requestBuffer) => {
+            const intersection = await findIntersect(
+              points || [await createPointFromCurrentTip({ socket, closeOnCompletion: false })],
+              {
+                socket,
+                closeOnCompletion: false
+              }
+            )
+            ensureSocketIsOpen(socket)
+            for (let n = 0; n <= (requestBuffer || 1000); n += 1) {
```

Comment:
either `let n = 1` or `n < (requestBuffer || 1000)`, otherwise it's off by one :s

Reply:
I'll update it here too
https://github.com/KtorZ/cardano-ogmios/blob/47ac39dbbaf2869d27ba09e9361bab593775f0ff/server/static/tests/chain-sync.js#L84